docs/src/pages/[platform]/connected-components/authenticator/headless/advanced/example.angular.mdx

```diff
@@ -8,18 +8,43 @@ import { AuthenticatorService } from '@aws-amplify/ui-angular';
 import awsExports from './aws-exports';
 
 @Component({
-  selector: 'app-root',
-  templateUrl: 'app.component.html',
+  selector: 'use-authenticator',
+  templateUrl: 'useAuthenticator.component.html',
 })
 export class UseAuthenticatorComponent {
   constructor(public authenticator: AuthenticatorService) {
     Amplify.configure(awsExports);
   }
 }
+
 ```
 
-_app.component.html_
+_use-authenticator.component.html_
 
-```js file=../../../../../../../../examples/angular/src/pages/ui/components/authenticator/useAuthenticator/useAuthenticator.component.html#L1-L8
 
+```html{4-6} 
```

Comment:
If we hardcode the examples here instead of including them from examples folder, how will they get tested?

Reply:
It won't. We don't have an exact github example that matches this scenario, really. It's similar.